Phonebook.Frontend/src/app/app.component.ts

```diff
@@ -95,8 +99,19 @@ export class AppComponent implements OnInit {
     //   });
     // }
 
+    //if skip_cookies is set, dont show dialogs
+    if (this.skipDialog) {
+      this.store.dispatch(new SetDisplayedNotificationVersion(DisplayNotificationDialog.version));
+    }
+    //subscribe on query param changes, if changed open snackbar
+    this.activatedRoute.queryParamMap.pipe(untilComponentDestroyed(this)).subscribe(queryParamMap => {
+      if (queryParamMap.get('skip_dialog') === 'true') {
+        this.skipDialog = true;
+        this.openSkippedDialogsSnackBar();
```

Comment:
Can we have two different Notifications? If you added the URL as favourite you keep gettings the call to action. 

I would propose to have two different messages: 
1. Speicher die Seiten-URL jetzt als Favorit um Startdialoge auch zukünftig zu überspringen. Bitte beachte: Mit dem gesetzten Url-Parameter werden keine Dialoge zu Updates oder Neuen Features angezeigt.
2. Dialoge sind deaktiviert. Bitte beachte das du keine Informationen zu Updates oder Neuen Features angezeigt bekommst.